app/models/step_by_step_page.rb

```diff
@@ -26,6 +26,10 @@ class StepByStepPage < ApplicationRecord
 
   scope :by_title, -> { order(:title) }
 
+  def approved_2i?
+    status.approved_2i?
```

Comment:
I think elsewhere we're just calling `step_by_step_page.status.approved_2i?` directly - not sure a helper method is necessary. If it is, could you explain why in the commit?